changelogs/fragments/ec2_key-fix-security-vulnerability.yml

```diff
@@ -0,0 +1,3 @@
+---
+breaking_changes:
+  - ec2_key - add support for new parameter `file_name` to save private key in when new key is created by AWS. When this option is provided the generated private key will be removed from the module return (https://github.com/ansible-collections/amazon.aws/pull/1704).
```

Comment:
```suggestion
  - ec2_key - add support for new parameter ``file_name`` to save private key in when new key is created by AWS. When this option is provided the generated private key will be removed from the module return (https://github.com/ansible-collections/amazon.aws/pull/1704).
```

Single backticks will try to create a link rather than a "code" block.